src/js/vue-plugins/document-utils.js

```diff
@@ -343,7 +343,11 @@ export default function install(Vue) {
       },
 
       getDocumentsBbox(documents) {
-        // given an array of documents, return the extent containing all these documents
+        documents = documents.filter((document) => document?.geometry?.geom);
```

Comment:
`document` can be undefined too? maybe you are using too many question marks there 🙂